core/filters.py

```diff
@@ -43,18 +44,12 @@ class ProgramListFilterSet(django_filters.FilterSet):
     )
 
 
-    def __init__(self, data, *args, **kwargs):
-        data = data.copy()
+    def __init__(self, data=None, *args, **kwargs):
+        data = data.copy() if data else None
```

Comment:
It doesn't look like we're using `data` except to pass it to the `super()` function. This copy is probably not needed.

Reply:
Deleted the copy requested here.